crates/apollo-smith/src/argument.rs

```diff
@@ -102,7 +103,7 @@ impl<'a> DocumentBuilder<'a> {
     /// Create an arbitrary `Argument`
     pub fn argument(&mut self) -> ArbitraryResult<Argument> {
         let name = self.name()?;
-        let value = self.input_value()?;
+        let value = self.input_value(Constness::NonConst)?;
```

Comment:
Arguments should only take `const` values when applied on schema elements, so this should be passed in from outside

Reply:
I've added it to the signature for `argument(s)` although we don't actually use them right now. I think we'll need to take a step back and look at the design again before we add actually use them as we will need to ensure that parameters are added in a consistent way across applications.

Reply:
oh i see!